src/main/java/org/mondemand/Context.java

```diff
@@ -86,9 +86,9 @@ public boolean equals(Object obj) {
     return true;
   }
 
-  public void addContext(Client client)
```

Comment:
This change technically breaks backwards compatibility, which would mean a major version bump